js/google_ima.js

```diff
@@ -1264,6 +1297,8 @@ require("../html5-common/js/utils/utils.js");
         {
           //destroy the current ad manager is there is one
           _IMA_SDK_destroyAdsManager();
+          this.currentIMAAd = null;
```

Comment:
We need to keep the currentIMAAd objects long enough to ensure a full cleanup. One such instance of this is in this.cancelAd, where we need these two for _endCurrentAd to cleanup properly.